package.json

```diff
@@ -23,31 +29,38 @@
   "author": "kiwi.com",
   "license": "MIT",
   "dependencies": {
+
+  },
+  "peerDependencies": {
     "react": "16.6.1",
-    "react-art": "16.6.1",
-    "react-dom": "16.6.1",
-    "react-native": "^0.57.5",
```

Comment:
having this in dependencies will cause crash for consumer, saying duplicate import of react-native

Reply:
Thank with these libraries I wasn't been sure where it belongs